metatron-app/metatron-discovery
# F#1224 unify dataprep preview codes

### Description

The server-side code related to the all kinds of previews has been unified.

**Related Issue** : 

#1224 

### How Has This Been Tested?

Test upload preview
1. Menu> Data Preparation > Dataset
2. Click "Generate new dataset"
3. Select File type
4. Import File ( excel, csv, json )
5. Check preview

### Types of changes



### Checklist:



### Additional Context
